locksmith/package.json

```diff
@@ -23,12 +23,9 @@
     "authorize": "./scripts/authorize-lock"
   },
   "lint-staged": {
-    "linters": {
-      "*.{js,ts}": [
-        "eslint --fix",
-        "git add"
-      ]
-    }
+    "*.{js,ts}": [
```

Comment:
For each file type, we define the scripts to be called.
`git add` is added by default.